css/port.css

```diff
@@ -104,44 +111,135 @@ a {
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
+  padding-top: 25px;
+}
+
+.section_header {
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+  color: #091e42;
+}
+
+.section_text {
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+}
+
+/* media query for the header  */
+@media (max-width: 470px) {
+  .nav {
+    width: 50%;
+    font-size: 15px;
+    display: none;
+  }
+
+  .menu-icon {
```

Comment:
- [ ] Kindly make sure you use both Flexbox and Grid in order to layout the elements on the page. You have used flexbox so kindly use grid for both mobile and desktop version.